ui/components/app/multichain-transaction-details-modal/multichain-transaction-details-modal.tsx

```diff
@@ -0,0 +1,365 @@
+
+  return (
+    <Modal
+      onClose={onClose}
+      data-testid="multichain-transaction-details-modal"
+      isOpen
+      isClosedOnOutsideClick={true}
+      isClosedOnEscapeKey={true}
+    >
+      <ModalOverlay />
+      <ModalContent
+        modalDialogProps={{
+          display: Display.Flex,
+          flexDirection: FlexDirection.Column,
+          padding: 4,
+        }}
+      >
+        <ModalHeader onClose={onClose} padding={0}>
+          <Text variant={TextVariant.headingMd} textAlign={TextAlign.Center}>
+            {capitalize(transaction.type)}
+          </Text>
+          <Text
+            variant={TextVariant.bodyMd}
+            color={TextColor.textAlternative}
+            textAlign={TextAlign.Center}
+          >
+            {transaction.timestamp
+              ? `${formatDateWithYearContext(
+                  transaction.timestamp,
+                  'MMM d, y',
+                  'MMM d',
+                )}, ${new Date(transaction.timestamp)
+                  .toLocaleTimeString('en-US', {
+                    hour: '2-digit',
+                    minute: '2-digit',
+                    hour12: false,
+                    timeZone: undefined,
+                  })
+                  .slice(0, 5)}`
+              : ''}
+          </Text>
+        </ModalHeader>
+
+        <Box paddingBottom={4}>
+          <Divider />
+        </Box>
+        <Box>
+          <Box
+            display={Display.Flex}
+            flexDirection={FlexDirection.Column}
+            gap={3}
+          >
+            {/* Status */}
+            <Box
+              display={Display.Flex}
+              justifyContent={JustifyContent.spaceBetween}
+            >
+              <Text variant={TextVariant.bodyMd} fontWeight={FontWeight.Medium}>
+                {t('status')}
+              </Text>
+              <Text
+                variant={TextVariant.bodyMd}
+                color={getStatusColor(transaction.status)}
+              >
+                {capitalize(transaction.status)}
+              </Text>
+            </Box>
+
+            {/* Transaction ID */}
+            <Box
+              display={Display.Flex}
+              justifyContent={JustifyContent.spaceBetween}
+            >
+              <Text variant={TextVariant.bodyMd} fontWeight={FontWeight.Medium}>
+                {t('notificationItemTransactionId')}
+              </Text>
+              <Box
+                display={Display.Flex}
+                alignItems={AlignItems.center}
+                gap={1}
+              >
+                <ButtonLink
+                  size={ButtonLinkSize.Inherit}
+                  textProps={{
+                    variant: TextVariant.bodyMd,
+                    alignItems: AlignItems.flexStart,
+                  }}
+                  as="a"
+                  externalLink
+                  href={`${blockExplorerUrl}/tx/${transaction.id}`}
+                >
+                  {shortenAddress(transaction.id)}
```

Comment:
Just to make it a bit more explicit, I'd make a local function `shortenTransactionId` instead. We could re-use `shortenAddress` for now, but adding a comment explaining that we use a similar output for now, WDYT?